### ISSUE TITLE
[Feature Request] UX improvement on the buttons in typing field.

### ISSUE THREAD

**Issue by aemxn** [telegramdesktop/tdesktop#7808]
### Is your feature request related to a problem?
Not a problem per se, but rather a UX improvements when typing a message.

To put it simply, when using a mouse to navigate on the screen, we sometimes accidentally hovered on the emoji button which makes it pop a dialog message containing emojis/stickers and whatnot. When our initial intention was just to press the send message button, this popup is kinda distracting and sometimes when miss-clicked, will send wrong message (stickers, gif, emoji).

![ux1][ux1]

### Describe the solution you'd like

Replace the whole button to show an **arrow** instead, which when clicked, it will bring out the buttons. This will not only reduces space, but will improve greatly on users focus on writing message.

![ux2][ux2]

[ux1]:https://i.imgur.com/BVhUG9K.png
[ux2]: https://i.imgur.com/6i0lgu0.jpg

**Comment by ilya-fedin** [comment]
#1333? :thinking:

**Comment by martagolimowska** [comment]
Cghdgjhchj

ilya-fedin <notifications@github.com> schrieb am Mi., 6. Mai 2020, 05:08:

**Comment by Archydos** [comment]
Why hasn't this issue been dealt with? It's been ongoing since 2015 if I'm not mistaken.

**Comment by aemxn** [comment]
@Archydos maybe it's not as important or there's a conflicting rules on Telegram Desktop UX.

It's not an issue if you are using a maximized window, but for some people (like me) always like to use the smallest window to use Telegram Desktop.

**Maximized** -- fixed:

![Imgur](https://i.imgur.com/NJAObfj.png)

**Smallest window** -- popped up:

![Imgur](https://i.imgur.com/PBfTXOc.png)

**Comment by reallyuniquename** [comment]
Still valid.

**Comment by aemxn** [comment]
Updated first post with a gif.

**Comment by bn-l** [comment]
What I love about this issue (still not fixed since 2015--the issue was simply closed) is that when you're typing and your mouse moves accidentally over the fucking emoji button, if you move it away and then back a bit too soon the window will reappear.

This is because the element goes stays transparent for a while (still taking hover events) before actually disappearing. If you're distracted you can do this a few times without realizing what's going on fully.

6 years. Can we at least have an option to remove this button?

**Comment by reallyuniquename** [comment]
Related PR #8009.

**Comment by bn-l** [comment]
@john-preston HELLO!?!?!?!

**Comment by john-preston** [comment]
@bn-l Hi. How are you?

**Comment by ivmazurenko** [comment]
When the hovering will be removed?

With this hovering behaviour several times I have sent stickers accidentally... to working chats! 

Okay, maybe hovering is not so bad. But it is absolutely necessary to prevent sending random pictures to random peoples somehow. Maybe you can implement modal dialog with text:

```
"Are you want to send this sticker with balls to your boss?"

"Yes, I want to sent sticker with balls to my boss"          "No, this is bad idea"
```

**Comment by ilya-fedin** [comment]
> When the hovering will be removed?

A lot of times has been sais that this won't be removed (probably someone at high level proibits this to the developer), so the only way is to use forks that aren't controlled by Telegram.

**Comment by ivmazurenko** [comment]
But this is unexpected behaviour, therefore this is a bug

**Comment by ivmazurenko** [comment]
It is looks very strange. I want to copy the text of message. But, instead, telegram decides that i want to send sticker with human balls, naked lady, or some twerking animal to my colleagues.

I don't like to send penises or twerking animals to my colleagues. Is there any logic, why I must to do that?

**Comment by ilya-fedin** [comment]
> But this is unexpected behaviour, therefore this is a bug

That's really subjective

**Comment by ivmazurenko** [comment]
> > But this is unexpected behaviour, therefore this is a bug
> 
> That's really subjective

No. Did you send sometimes cherry, that shaking its butt to you colleagues?

**Comment by ilya-fedin** [comment]
>  I want to copy the text of message. But, instead, telegram decides that i want to send sticker with human balls, naked lady, or some twerking animal to my colleagues.

Since the popup appears before you right click at the message, I don't imagine how that could happen

**Comment by ilya-fedin** [comment]
> No.

Yeah, it is. You have your workflow, other have theirs workflow. It's disturbing for you, it's not disturbing for others. A bug is somewhat that is not intended. It's an intended behavior (according to @john-preston) even if it doesn't fit into your workflow.

**Comment by ivmazurenko** [comment]
So, add some kind of option to configurations ?

**Comment by ilya-fedin** [comment]
> So, add some kind of option to configurations ?

@john-preston follows "less options is better" paradigm, so that's no-go

**Comment by ivmazurenko** [comment]
One more time:
![114711998-e40ccd00-9d2f-11eb-9c8e-1b7c7400cca0](https://user-images.githubusercontent.com/34516127/116778664-eb481080-aa7b-11eb-82d1-6c149131cd07.gif)

**Comment by ilya-fedin** [comment]
You should get that even @john-preston doesn't have a free to change something. Entire interface is controlled by Telegram designers that give Preston a mockup that he must follow and can't do any changes. Telegram designers won't hear you here, that bugtracker is official only to @john-preston, but not for Telegram team and no one else read here.

**Comment by ilya-fedin** [comment]
> One more time:

I see you open that popup, then slowly move the mouse, then click the sticker for a reson. I don't know why you do that, but that's insane, you can just stop when it opens.

**Comment by ivmazurenko** [comment]
One more time

https://user-images.githubusercontent.com/34516127/116778812-e899eb00-aa7c-11eb-8254-9e93b91cb428.mov

**Comment by ivmazurenko** [comment]
One more time

**Comment by ivmazurenko** [comment]
No thanks, I will recreate such issues until Telegram command will not fixed it

**Comment by ilya-fedin** [comment]
Then you will be banned most likely :laughing:

**Comment by Aokromes** [comment]
> No thanks, I will recreate such issues until Telegram command will not fixed it

spamming issues doesn't helps to get fixed, more likely the oposite.

**Comment by ivmazurenko** [comment]
Hello, felons, today my colleague sends us a sticker with label "Анус - цена успеха". 

Thank you for hovering sticker panel

**Comment by servitola** [comment]
Totally helpful thread. I do this mistake all the time

**Comment by RememberTheAir** [comment]
Beta 3.5.3 has an option under `settings -> advanced -> experimental` to activate the emoji panel only on click
